frontend/src/config/rbd/translation.json

```diff
@@ -129,5 +129,6 @@
   "Phase classification": "Classification de la phase",
   "Population in phase 3 to 5": "Population en phase 3 à 5",
   "Days since last rain": "Les jours depuis la dernière pluie",
-  "Admin Level": "Niveau Administratif"
+  "Admin Level": "Niveau Administratif",
+  "Classification": "Classification"
```

Comment:
I acknowledge this is incredibly unnecessary, it's just coverage for coverage's sake — but it demonstrates that there's coverage to everyone who doesn't already know that "Classification" = "Classification"

Reply:
Yup I think it's the right approach